examples/magento-graphcms/pages/checkout/index.tsx

```diff
@@ -81,32 +91,22 @@ function ShippingPage() {
               <LayoutTitle icon={iconBox}>
                 <Trans id='Shipping' />
               </LayoutTitle>
-
-              <EmailForm step={1}>
-                <Typography
-                  variant='body2'
-                  component='ul'
-                  sx={(theme) => ({ pl: theme.spacings.xs, mt: theme.spacings.xxs })}
-                >
-                  <li>
-                    <Trans id='Email address of existing customers will be recognized, sign in is optional.' />
-                  </li>
-                  <li>
-                    <Trans id='Fill in password fields to create an account.' />
-                  </li>
-                  <li>
-                    <Trans id='Leave password field empty to order as guest.' />
-                  </li>
-                </Typography>
-              </EmailForm>
-
-              <ShippingAddressForm step={2} />
+              {addresses ? (
+                <CustomerAddressForm step={2}>
+                  <ShippingAddressForm ignoreCache step={3} />
```

Comment:
@paales So .. this is pretty much a DirtyFix™ (i think)
We want our form to be empty when we click the 'New address' option. I tried messing around with useFormPersist etc but could not get it to work properly. 

This works just fine, but I'm too hyped about introducing a new prop for this.